featuretools/utils/recommend_primitives.py

```diff
@@ -0,0 +1,261 @@
+from typing import List
+
+import pandas as pd
+
+from featuretools.computational_backends import calculate_feature_matrix
+from featuretools.entityset import EntitySet
+from featuretools.primitives.utils import get_transform_primitives
+from featuretools.synthesis import dfs, get_valid_primitives
+
+ORDERED_PRIMITIVES = [
+    "absolute_diff",
+    "cum_sum",
+    "cum_count",
+    "cum_mean",
+    "cum_max",
+    "cum_min",
+    "cumulative_time_since_last_false",  # time_index
+    "cumulative_time_since_last_true",  # time_index
+    "diff",
+    "diff_datetime",
+    "exponential_weighted_average",
+    "exponential_weighted_std",
+    "exponential_weighted_variance",
+    "greater_than_previous",
+    "is_first_occurrence",
+    "is_last_occurrence",
+    "is_max_so_far",
+    "is_min_so_far",
+    "lag",  # time_index
+    "less_than_previous",
+    "percent_change",
+    "same_as_previous",
+    "time_since_previous",  # time_index
+]
+
+
+DEPRECATED_PRIMITIVES = [
+    # // as of premium - primitives 0.14.0 and feature - tools 0.26.1
+    "multiply_boolean",  # functionality duplicated by 'and' primitive
+    "numeric_lag",  # deperecated and replaced with `lag`
+]
+
+REQUIRED_INPUT_PRIMITIVES = [
+    # // as of premium - primitives 0.14.0 and feature - tools 0.26.1
+    "count_string",
+    "distance_to_holiday",
+    "is_in_geobox",
+    "score_percentile",
+    "subtract_numeric_scalar",
+    "scalar_subtract_numeric_feature",
+    "not_equal_scalar",
+    "multiply_numeric_scalar",
+    "modulo_numeric_scalar",
+    "divide_numeric_scalar",
+    "add_numeric_scalar",
+    "equal_scalar",
+    "greater_than_equal_to_scalar",
+    "less_than_equal_to_scalar",
+    "divide_by_feature",
+    "greater_than_scalar",
+    "less_than_scalar",
+    "modulo_by_feature",
+    "time_since",
+    "savgol_filter",
+    "isin",
+    "numeric_bin",
+]
+
+OTHER_PRIMITIVES_TO_EXCLUDE = [
```

Comment:
this is a little mysterious to me, can we change the name or add a comment explaining why these primitives are excluded?

Reply:
Added a comment